roles/pulibrary.capistrano/files/libimages1.smb.credentials

```diff
@@ -0,0 +1,8 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Orangelight doesn't need files like this. Is it ok to remove or are other apps going to use this role?

Reply:
I think it's fine to remove them — Figgy has its own copies: https://github.com/pulibrary/princeton_ansible/tree/master/roles/pulibrary.figgy/files